plugins/processors/starlark/testdata/drop_fields_with_unexpected_type.star

```diff
@@ -0,0 +1,26 @@
+# Drop fields if they NOT contain values of an expected type.
+#
+# In this example we ignore fields with an unknown expected type and do not drop them.
+#
+# Example Input:
+# measurement,host=hostname a=1i,b=4.2,c=42.0,d="v3.14",e=true,f=23.0 1597255410000000000
+# measurement,host=hostname a=1i,b="somestring",c=42.0,d="v3.14",e=true,f=23.0 1597255410000000000
+#
+# Example Output:
+# measurement,host=hostname a=1i,b=4.2,c=42.0,d="v3.14",e=true,f=23.0 1597255410000000000
+# measurement,host=hostname a=1i,c=42.0,d="v3.14",e=true,f=23.0 1597255410000000000
+
+expected_type = {
+    "a": "int",
+    "b": "float",
+    "c": "float",
+    "d": "string",
+    "e": "bool"
+}
+    
+def apply(metric):
+    for k, v in metric.fields.items():
+        if type(v) != expected_type.get(k, type(v)):
+            metric.fields.pop(k)
```

Comment:
we have access to a logger now, let's add a log entry to this example that says that we're dropping it because it's the wrong type, and what the actual and expected types were.